pontoon/base/static/js/locale_project.js

```diff
@@ -0,0 +1,65 @@
+$(function() {
+
+  // Search entire project with Enter (and remove default search handler)
+  $('.part .menu .search-project input[type=search]').off('keyup.search').keydown(function (e) {
+    if (e.which === 13) {
+      e.stopPropagation();
+
+      var q = $(this).val(),
+          action = '';
+
+      if (q.length > 0) {
+        if ($('.locale .selector .language').length) {
+          var locale = $('.locale .selector .language').attr('class').split(' ')[1],
```

Comment:
In general it's a bad idea to store actual data in classnames or other non-data attributes, because the form of your data should not have the side-effect on how the HTML is displayed or functions.